pages/common/dexdump.md

```diff
@@ -1,7 +1,7 @@
 # dexdump
 
 > Display information about Android DEX files.
-> More information: <https://manpages.ubuntu.com/manpages/latest/en/man1/dexdump.1.html>.
+> More information: <https://manned.org/dexdump>.
```

Comment:
OMG this is hilarious.  From the Ubuntu man page:
>### NAME
>```
>dexdump - Dex Tool
>```
>### DESCRIPTION
>```
>debian/out/dexdump:  error  while  loading shared libraries: libsigchain.so.0: cannot open
>shared object file: No such file or directory
>```
The manned.org one is okay though.